web/docs/project/env-vars.md

```diff
@@ -2,61 +2,165 @@
+Client environment variables are injected into the client Javascript code during the build process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys, you should store secrets in the server env variables).
 
-To enable Wasp to pick them up, client env vars must be prefixed with `REACT_APP_`, for example: `REACT_APP_SOME_VAR_NAME=...`.
+<ClientEnvVarsNote />
 
 You can read them from the client code like this:
 
 <Tabs groupId="js-ts">
 <TabItem value="js" label="JavaScript">
 
 ```js title="src/App.js"
-console.log(import.meta.env.REACT_APP_SOME_VAR_NAME)
+import { env } from 'wasp/client'
+
+console.log(env.REACT_APP_SOME_VAR_NAME)
 ```
 </TabItem>
 <TabItem value="ts" label="TypeScript">
 
 ```ts title="src/App.ts"
-console.log(import.meta.env.REACT_APP_SOME_VAR_NAME)
+import { env } from 'wasp/client'
+
+console.log(env.REACT_APP_SOME_VAR_NAME)
 ```
 </TabItem>
 </Tabs>
 
+Read more about the `env` object in the [API reference](#client-env-vars-1).
 
-Check below on how to define them.
+### Wasp Client Env Vars
 
+Here are the client env vars that Wasp defines:
+
+#### General Configuration
+
+<EnvVarsTable>
+  <EnvVar name="REACT_APP_API_URL" type="URL" isRequired={false} defaultValue="http://localhost:3001" note="The client uses this as the server URL." />
+</EnvVarsTable>
 
 ## Server Env Vars
 
-In server environment variables, you can store secret values (e.g. secret API keys) since they are not publicly readable. You can define them without any special prefix, such as `SOME_VAR_NAME=...`.
+You can store secret values (e.g. secret API keys) in the server env variables since they are not publicly readable. You can define them without any special prefix, such as `SOME_VAR_NAME=...`.
 
 You can read the env vars from server code like this:
 <Tabs groupId="js-ts">
 <TabItem value="js" label="JavaScript">
 
 ```js
-console.log(process.env.SOME_VAR_NAME)
+import { env } from 'wasp/server'
+
+console.log(env.SOME_VAR_NAME)
 ```
 </TabItem>
 <TabItem value="ts" label="TypeScript">
 
 ```ts
-console.log(process.env.SOME_VAR_NAME)
+import { env } from 'wasp/server'
+
+console.log(env.SOME_VAR_NAME)
 ```
 </TabItem>
 </Tabs>
 
-Check below on how to define them.
+Read more about the `env` object in the [API reference](#server-env-vars-1).
+
+### Wasp Server Env Vars
+
+#### General Configuration
+
+<EnvVarsTable>
+    <EnvVar name="DATABASE_URL" type="String" isRequired={true} note="The URL of the PostgreSQL database you want your app to use." />
+    <EnvVar name="WASP_WEB_CLIENT_URL" type="URL" isRequired={true} note="Used as your client URL. The server needs to know about it to properly configure Same-Origin Policy (CORS) headers." />
```

Comment:
And some other stuff I think, for example provide a link to the client in some automatic emails (auth)? Maybe not any more if flow is completely server-side, but I would maybe state is to that it sound slike CORS is not or might not be the only thing this is important for. I wonder if we should even explain why server needs it -> that info can easily become obsolete, maybe better to skip it? Not a biggie though.